src/Sql/dbo/Stored Procedures/OrganizationUser_ReadManyResetPasswordDetailsByOrganizationUserIds.sql

```diff
@@ -0,0 +1,24 @@
+﻿CREATE PROCEDURE [dbo].[OrganizationUser_ReadManyResetPasswordDetailsByOrganizationUserIds]
```

Comment:
Similar to my comment about naming things in the controller and repository layers: this stored procedure name should use "account recovery" verbiage instead of the outdated "reset password". We can't do anything about the column right now, but that should be included in the tech debt epic.